packages/server-express/src/middlewares/session-middleware.ts

```diff
@@ -0,0 +1,92 @@
+import type { TSessionMiddlewareOptions } from '../types';
+import type { Request, Response, NextFunction } from 'express';
+
+import { CLOUD_IDENTIFIERS } from '../constants';
+import expressJwtMiddleware from 'express-jwt';
+import jwksRsa from 'jwks-rsa';
+
+const decodedTokenKey = 'decoded_token';
+
+const mapCloudIdentifierToUrl = (
+  issuer: TSessionMiddlewareOptions['issuer']
+) => {
+  switch (issuer) {
```

Comment:
Sorry, got confused here. We map the cloud identifier to an url but we use the issuer to do so. Should we name the issuer just cloudIdentifier?

Reply:
I went back and forth with this and eventually settled on this: the `issuer` is a required field, and you can pass one of the identifiers. However you can also "bail out" and pass the URL yourself.
Initially I had it split into two fields, but both are then optional and it just gets complicated for validation and stuff.

I'm all hears if you have other suggestions.